device-types/Netgate/8300-Security-Gateway.yaml

```diff
@@ -0,0 +1,53 @@
+---
+manufacturer: Netgate
+model: 8300 Security Gateway
+slug: netgate-8300-security-gateway
+u_height: 1
+is_full_depth: false
+airflow: front-to-rear
+console-server-ports:
```

Comment:
This is not correct use of console-server-ports and so should be removed. "A console server is a device which provides remote access to the local consoles of connected devices. They are typically used to provide remote out-of-band access to network devices, and generally connect to console ports."